test/Log/LogServiceTest.php

```diff
@@ -218,26 +217,11 @@ public function testGetDefaultFormatNotConfigured()
 
         $format = $logService->getDefaultFormat();
 
-        $expected = preg_replace('/%runId%/', $logService->getRunId(), LogService::DEFAULT_FORMAT);
-        $expected .= PHP_EOL;
+        $expected = LogService::DEFAULT_FORMAT;
 
         $this->assertEquals($expected, $format);
     }
 
-    /**
-     * Format should contain run ID.
-     */
-    public function testGetDefaultFormatContainsRunId()
```

Comment:
It makes sense to remove that test (and the other one). However I think a couple of new tests should be added. We have a new function. And we have a function with additional behavior now (`createLogger`). Testing protected functions is possible and could be debated. My question is, how do I know that %runId% is being used in the format? Would any test break if you comment out that line?

Reply:
Not using `prepareFormat()` in `createLogger()` won't break any test. I will add a test for `prepareFormat()`, this will let us know that `%runId% is being replaced. Then we won't have to get into testing protected function.